backend/danswer/one_shot_answer/answer_question.py

```diff
@@ -204,12 +213,17 @@ def stream_answer_objects(
         # tested quotes with tool calling too much yet
         skip_explicit_tool_calling=True,
         return_contexts=query_req.return_contexts,
+        skip_generation=True,
```

Comment:
I'm pretty sure this breaks the other APIs that rely on this (and that do give back an answer)
Also this breaks the SlackBot no?

Reply:
For some reason I was under the false pretension that one_shot's `answer_question` was only used by the search endpoint ad moved this flag from the request to a default. Tested the Slack bot but seems like I happened to have been on a different branch. Will move to the fetch request and include as an attribute to `DirectQARequest` like `evaluate_response` is currntly